src/fshtypes/FshCodeSystem.ts

```diff
@@ -26,51 +23,6 @@ export class FshCodeSystem extends FshEntity {
     return 'FshCodeSystem';
   }
 
-  addConcept(newConcept: ConceptRule) {
-    if (this.checkConcept(newConcept)) {
-      this.rules.push(newConcept);
-    }
-  }
-
-  checkConcept(newConcept: ConceptRule): boolean {
-    const existingConcept = this.rules.find(
-      rule => rule instanceof ConceptRule && rule.code == newConcept.code
-    ) as ConceptRule;
-    if (existingConcept) {
-      // if the new ConceptRule has only a code and a hierarchy,
-      // and the existing ConceptRule with that code has a matching hierarchy,
-      // the user may simply be referencing the existing concept to establish context.
-      if (
-        newConcept.display == null &&
-        newConcept.definition == null &&
-        isEqual(existingConcept.hierarchy, newConcept.hierarchy)
-      ) {
-        return false;
-      } else {
-        throw new CodeSystemDuplicateCodeError(this.id, newConcept.code);
-      }
-    }
-    // check the hierarchy, if applicable
-    // for each predecessor element in the new concept's hierarchy, we should be able to find a rule that
-    // 1. defines a concept
-    // 2. with the predecessor's code
-    // 3. and a hierarchy equal to everything before it in the new concept's hierarchy
-    newConcept.hierarchy.forEach((predecessor, i) => {
-      if (
-        !this.rules.some(rule => {
-          return (
-            rule instanceof ConceptRule &&
-            rule.code === predecessor &&
-            isEqual(rule.hierarchy, newConcept.hierarchy.slice(0, i))
-          );
-        })
-      ) {
-        throw new CodeSystemIncorrectHierarchyError(this.id, newConcept.code);
```

Comment:
We're not using `CodeSystemIncorrectHierarchyError` anymore, so we can delete that class.